.github/workflows/cpp-packaging.yml

```diff
@@ -37,6 +37,7 @@ env:
   xcodeVersion: "12.2"
   # LLVM version with ARM MachO support has no version number yet.
   llvmVer: "5f187f0afaad33013ba03454c4749d99b1362534"
+  GITHUB_TOKEN: ${{ github.token }}
```

Comment:
FYI this line is here to ensure that CMake gets the token

Reply:
Would it need to be added to the other workflows?